.github/workflows/handle-new-contribution.yml

```diff
@@ -0,0 +1,44 @@
+name: Handle New Contribution
+
+on:
+  pull_request_target:
+    types: [opened, ready_for_review]
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  assign_and_label:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.fork == true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - name: Print Context
+        run: |
+          echo "$GITHUB_CONTEXT"
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+      - name: Add an assignee and a Contribution label
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          PR_URL: ${{github.event.pull_request.html_url}}
+        run: |
+          echo "get content_roles.json using curl"
+          contrib_tl=$( curl https://raw.githubusercontent.com/demisto/content/master/.github/content_roles.json | jq .CONTRIBUTION_TL -r )
+          echo "$contrib_tl"
+          echo "Finished retrieving the contribution TL username"
+          
+          if [ $contrib_tl ]; then
```

Comment:
does this work? we dont need -z or -n?

Reply:
yes, it works, I checked in the related test PR. Actually, when I tried with -z it didn't work. I took it from [here](https://serverfault.com/questions/7503/how-to-determine-if-a-bash-variable-is-empty).
![image](https://github.com/demisto/dockerfiles/assets/71597826/8a9f5608-4c1a-487b-a876-31f5f68e8260)